airbyte-integrations/connectors/source-mongodb-v2/README.md

```diff
@@ -1,5 +1,27 @@
 # MongoDb Source
 
+## Schema
+
+For each collection, if you want to define a schema in order to propagate type to Glue, specify the schema for the collection as a unique string. 
```

Comment:
The schema definition here shouldn't be unique or specific to Glue.... sources shouldn't know about destinations

Reply:
True true removed from the readme

Reply:
I'll see if we can avoid this by changing the spec schema